Cargo.toml

```diff
@@ -22,10 +22,10 @@ lto = "thin"
 mbedtls = { git = "https://github.com/mobilecoinfoundation/rust-mbedtls.git", rev = "6d8fe323a3292f87a6bce4b35963d47139a583f9" }
 mbedtls-sys-auto = { git = "https://github.com/mobilecoinfoundation/rust-mbedtls.git", rev = "6d8fe323a3292f87a6bce4b35963d47139a583f9" }
 
-mc-sgx-dcap-types = { git = "https://github.com/mobilecoinfoundation/sgx.git", rev = "33998eb12e7b13a963519fcb202413a5643abdf9" }
-mc-sgx-core-sys-types = { git = "https://github.com/mobilecoinfoundation/sgx.git", rev = "33998eb12e7b13a963519fcb202413a5643abdf9" }
-mc-sgx-core-types = { git = "https://github.com/mobilecoinfoundation/sgx.git", rev = "33998eb12e7b13a963519fcb202413a5643abdf9" }
-mc-sgx-dcap-sys-types = { git = "https://github.com/mobilecoinfoundation/sgx.git", rev = "33998eb12e7b13a963519fcb202413a5643abdf9" }
+mc-sgx-dcap-types = { git = "https://github.com/mobilecoinfoundation/sgx.git", rev = "267762164a219245f0cbf920a03f78ef38204b21" }
```

Comment:
Similar to #170, this needs to wait for https://github.com/mobilecoinfoundation/sgx/pull/419 and a subsequent release before being merged

Reply:
no longer applies